dbt_project.yml

```diff
@@ -1,24 +1,27 @@
 config-version: 2
 
 name: 'zendesk_source'
-version: '0.0.1'
+version: '0.1.2'
 
-require-dbt-version: [">=0.18.0", "<0.19.0"]
+require-dbt-version: [">=0.18.0", "<0.20.0"]
 
 models:
-    zendesk_source:
-        materialized: table
+  zendesk_source:
+    materialized: table
+    tmp:
+      materialized: view
 
 vars:
-    zendesk_source:
-        group: "{{ source('zendesk', 'group') }}"
-        organization: "{{ source('zendesk', 'organization') }}"
-        schedule: "{{ source('zendesk', 'schedule') }}"
-        ticket: "{{ source('zendesk', 'ticket') }}"
-        ticket_comment: "{{ source('zendesk', 'ticket_comment') }}"
-        ticket_field_history: "{{ source('zendesk', 'ticket_field_history') }}"
-        ticket_schedule: "{{ source('zendesk', 'ticket_schedule') }}"
-        ticket_tag: "{{ source('zendesk', 'ticket_tag') }}"
-        user: "{{ source('zendesk', 'user') }}"
-        using_sla_policy: True
-        using_schedules: True
+  zendesk_source:
+    group:                "{{ source('zendesk', 'group') }}"
+    organization:         "{{ source('zendesk', 'organization') }}"
+    schedule:             "{{ source('zendesk', 'schedule') }}"
+    ticket:               "{{ source('zendesk', 'ticket') }}"
+    ticket_comment:       "{{ source('zendesk', 'ticket_comment') }}"
+    ticket_field_history: "{{ source('zendesk', 'ticket_field_history') }}"
+    ticket_schedule:      "{{ source('zendesk', 'ticket_schedule') }}"
+    ticket_tag:           "{{ source('zendesk', 'ticket_tag') }}"
+    user:                 "{{ source('zendesk', 'user') }}"
+
+    using_sla_policy: True
```

Comment:
@fivetran-kristin I wasn't able to find this variable `using_sla_policy` set as the config to enable any models or add/remove sla fields in this package or the transform package. Is this variable still necessary?

Reply:
Great catch! The first iteration of this package used an sla-specific table which ended up not being necessary so it was removed. Turns out this is leftover from that and should be deleted. Thanks!

Reply:
Updated!